include/wlr/util/direction.h

```diff
@@ -0,0 +1,11 @@
+#ifndef WLR_UTIL_DIRECTION_H
+#define WLR_UTIL_DIRECTION_H
+
+enum wlr_direction {
+	WLR_DIRECTION_UP = 0,
+	WLR_DIRECTION_DOWN = 1,
+	WLR_DIRECTION_LEFT = 2,
+	WLR_DIRECTION_RIGHT = 4,
+};
```

Comment:
I don't think this merits its own header

Reply:
Where do you want it?

Reply:
Can just live in wlr_output_layout.h imo